index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio setup and mobile first</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header1">
+
+        <h3 class="logo">My Logo</h3>
+        <a href="#"><img src="./img/hamburger.png" alt="hamburger_image"></a>
+
+    </header>
+
+    <section class="section1">
+
+        <div class="intro">
+
+            <h1>I'm Emeka Glad to see you!</h1>
+
+            <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work
+                and experience! if you like what you see and have a project you need coded, don't hesistate to contact
+                me.
+            </p>
+
+        </div>
+
+        <div class="footer1">
+
```

Comment:
[OPTIONAL] 👍🏽 

- I think it will be better if you can re name your `class="footer1"`  to something that references what this section is all about. Apparently, this is because this section of your project is not the footer. 🍷